Mathlib/LinearAlgebra/BilinearForm/Basic.lean

```diff
@@ -43,326 +44,239 @@ In this file we use the following type variables:
 Bilinear form,
 -/
 
```

Comment:
```suggestion

@[deprecated] --  2024-04-08
abbrev BilinForm (R M : Type*) [CommSemiring R] [AddCommMonoid M] [Module R M] := M →ₗ[R] M →ₗ[R] R

```

Reply:
@ocfnash  I tried that before, but it seems to conflict with `open LinearMap (BilinForm)` https://github.com/leanprover-community/mathlib4/pull/11278#discussion_r1536821607 so we abandoned that idea.

Reply:
Fair enough.